homeassistant/components/alert/__init__.py

```diff
@@ -47,6 +49,7 @@
         ),
         vol.Optional(CONF_CAN_ACK, default=DEFAULT_CAN_ACK): cv.boolean,
         vol.Optional(CONF_SKIP_FIRST, default=DEFAULT_SKIP_FIRST): cv.boolean,
+        vol.Optional(CONF_THROTTLE, default=DEFAULT_THROTTLE): vol.Coerce(float),
```

Comment:
We don't allow changes to YAML integrations that are YAML only at this point; especially because these could be migrated to the UI.

However, with that said, development on this integration is currently frozen, and changes for it should not be processed, as the functionality of this integration can be achieved using an automation or automation blueprint without the need for any integration at all.

We currently keep the integration around for historical/backward compatibility purposes, but are not planning on modernizing, refactoring, or extending it.

Nevertheless, thanks for being willing to contribute 👍 

../Frenck